engine/src/engine.rs

```diff
@@ -860,7 +861,12 @@ pub fn submit<I: IO + Copy, E: Env, P: PromiseHandler>(
         .address
         .ok_or(EngineErrorKind::InvalidSignature)?;
 
-    sdk::log!(crate::prelude::format!("signer_address {:?}", sender).as_str());
+    sdk::log!(crate::prelude::format!(
+        "tx_hash: 0x{}",
+        hex::encode(aurora_engine_sdk::keccak(transaction_bytes))
```

Comment:
I don't think this is a good change. We can see by the minor bump in the regression tests that the cost is non-zero and I don't think any value is added by logging this. This hashing could be done by looking at the data in the NEAR transaction directly in whatever offline analysis we are doing, no need to do it in the engine.

Reply:
@joshuajbouw

Reply:
Yeah, good point. Lets close this for now.